src/hackerspace_online/tests/test_management_commands.py

```diff
@@ -128,3 +131,52 @@ def test_added_rows_to_db(self):
         self.assertEqual(Quest.objects.count(), expected_quest_count)
         self.assertEqual(Category.objects.count(), expected_campaign_count)
         self.assertEqual(User.objects.count(), expected_user_count)
+
+
+class FullCleanTest(TestCase, CommandMixin):
+    name = 'full_clean'
+
+    def setUp(self):
+        call_command('initdb')
+
+        # have to create tenant this way to prevent errors
+        # + "you cant create a tenant outside public schema"
+        with schema_context(get_public_schema_name()):
+            self.tenant1 = Tenant.objects.create(schema_name='test_schema1', name='Test Tenant 1')
+        with schema_context(get_public_schema_name()):
+            self.tenant2 = Tenant.objects.create(schema_name='test_schema2', name='Test Tenant 2')
+
+        # add two different errors to each tenant
+        with schema_context(self.tenant1.schema_name):
+            # will cause an error because author is None because of
+            # `null=True` without `blank=True`
+            baker.make('announcements.Announcement')
+
+        with schema_context(self.tenant2.schema_name):
+            # will cause an error because semester is None because of
+            # `null=True` without `blank=True`
+            baker.make('quest_manager.QuestSubmission')
+
+    def test_full_clean(self):
+        """ Checks if full clean captures expected validation errors from "full_clean" management command
+        See setUp for the expected errors.
```

Comment:
Why are the things you are testing "expected" validation errors?

" # profile is here because of grad_year being empty for staff"  <-- more context please.

Reply:
done